src/loaders/ObjectLoader.js

```diff
@@ -610,6 +617,94 @@ THREE.ObjectLoader.prototype = {
 
 			}
 
+			/*
+			 * Now SkinnedMesh could have two bone sets in its children,
+			 * one is made from geometry.bones in SkinnedMesh constroctor
+			 * and another one is made from data.children in this method.
+			 * These two bone sets correspond to the same bones.
+			 * The former would represent a default stand pose while
+			 * the latter would represent a pose updated by skinning.
+			 * To let SkinnedMesh be updated pose, here this logic copies
+			 * the latter one to the former one then removes the latter one.
+			 * If in the latter one there're objects which don't correspond to
+			 * the ones in the former one, they'll be moved to the former one.
+			 */
```

Comment:
So let me make the issue more specific with an example.

Imagine that there is a geometry with bones which
hold default pose bone parameters as just Objects.

    // Bone B is a child of Bone A
    geometry
        - bones[0] (Object A)
        - bones[1] (Object B)
        - bones[2] (Object C)

You create SkinnedMesh instance from that geometry.
Bone instances are generated in SkinnedMesh constructor
and they'll be children of SkinnedMesh with forming of parent-child structure
and also will be bones of SkinnesMesh.skeleton.

    SkinnedMesh
        - geometry
            - bones[0] (Object A)
            - bones[1] (Object B)
            - bones[2] (Object C)
        - skeleton
            - bones[0] (Bone A)
            - bones[1] (Bone B)
            - bones[2] (Bone C)
        - children[0] (Bone A)
            - children[0] (Bone B)
        - children[1] (Bone C)

You add Mesh to Bone B.

    SkinnedMesh
        - geometry
            - bones[0] (Object A)
            - bones[1] (Object B)
            - bones[2] (Object C)
        - skeleton
            - bones[0] (Bone A)
            - bones[1] (Bone B)
            - bones[2] (Bone C)
        - children[0] (Bone A)
            - children[0] (Bone B)
                - Mesh
        - children[1] (Bone C)

You let SkinnedMesh do skinning.

    // ' indicates updated instance by skinning
    SkinnedMesh
        - geometry
            - bones[0] (Object A)
            - bones[1] (Object B)
            - bones[2] (Object C)
        - skeleton
            - bones[0] (Bone A')
            - bones[1] (Bone B')
            - bones[2] (Bone C')
        - children[0] (Bone A')
            - children[0] (Bone B')
                - Mesh
        - children[1] (Bone C')

After that you create json with toJSON() and then 
re-create SkinnedMesh with ObjectLoader.parse(json).
(This represents auto-save and auto-load of Editor)

The issue that SkinnedMesh which you get has two bone sets.
One is made from geometry.bones and represents default pose (Bone A, B, C).
Another one is made from json and represents updated bones (Bone A', B', C')

    SkinnedMesh
        - geometry
            - bones[0] (Object A)
            - bones[1] (Object B)
            - bones[2] (Object C)
        - skeleton
            - bones[0] (Bone A)
            - bones[1] (Bone B)
            - bones[2] (Bone C)
        - children[0] (Bone A)
            - children[0] (Bone B)
        - children[1] (Bone C)
        - children[2] (Object3D A')
            - children[0] (Object3D B')
                - Mesh
        - children[3] (Object3D C')

So my idea is that copy and remove

    // SkinnedMesh.children[0] <- copy - SkinnedMesh.children[2]
    // SkinnedMesh.children[0][0] <- copy - SkinnedMesh.children[2][0]
    // SkinnedMesh.children[1] <- copy - SkinnedMesh.children[3]
    // move Mesh from a child of children[2][0] to a child of children[0][0]
    // remove SkinnedMesh.children[2], [2][0], [3]
    SkinnedMesh
        - geometry
            - bones[0] (Object A)
            - bones[1] (Object B)
            - bones[2] (Object C)
        - skeleton
            - bones[0] (Bone A')
            - bones[1] (Bone B')
            - bones[2] (Bone C')
        - children[0] (Bone A')
            - children[0] (Bone B')
                - Mesh
        - children[1] (Bone C')